lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers.go

```diff
@@ -89,11 +95,23 @@ var (
 	// Mobilityd states are keyed as <IMSI>.<APN>.
 	mobilitydStateKeyRe = regexp.MustCompile(`^(?P<imsi>IMSI\d+)\..+$`)
 
-	subscriberLoadCriteria       = configurator.EntityLoadCriteria{LoadMetadata: true, LoadConfig: true, LoadAssocsFromThis: true}
 	apnPolicyProfileLoadCriteria = configurator.EntityLoadCriteria{LoadAssocsFromThis: true, LoadAssocsToThis: true}
 )
 
-var subscriberStateTypes = []string{
+var subscriberStateTypesKeyedByIMSI = []string{
+	lte.ICMPStateType,
+	lte.MMEStateType,
+	lte.S1APStateType,
+	lte.SPGWStateType,
+	lte.SubscriberStateType,
+	orc8r.DirectoryRecordType,
+}
+
+var subscriberStateTypesKeyedByCompositeKey = []string{
+	lte.MobilitydStateType,
+}
+
+var allSubscriberStateTypes = []string{
```

Comment:
Is it possible to create this with a call to `append` ? To avoid duplication / accidental future bugs

If not, perhaps we create it within each function that needs it